app/brave_generated_resources.grd

```diff
@@ -1299,6 +1299,32 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
           Brave Player
         </message>
       </if> <!-- enable_brave_player -->
+
+      <!-- File select -->
+      <message name="IDS_FILE_SELECT_OPEN_TITLE" desc="">
+        <ph name="SITE">$1<ex>http://foo.com</ex></ph> wants to open
+      </message>
+      <message name="IDS_FILE_SELECT_OPEN_TITLE_IFRAME" desc="">
+        An embedded page at <ph name="SITE">$1<ex>http://foo.com</ex></ph> wants to open
+      </message>
```

Comment:
I am not certain about `at` here and the other similar strings. I want to say `on` would sound more expected.

Reply:
Thanks, done!